Algorithms/Problems/CoinChange/GreedyAlgorithm.cs

```diff
@@ -0,0 +1,81 @@
+﻿using System;
+using System.Linq.Expressions;
+using System.Security.Cryptography;
+using System.Text;
+
+public class GreedyAlgorithm
+
+    /*
+    // Info
+    //
+A greedy algorithm is the one that always chooses the best solution at the time, with no regard for how that choice will affect future choices. Here, we will discuss how to use Greedy algorithm to making coin changes.
```

Comment:
Codacy found an issue: [Split this 217 characters long line (which is greater than 200 authorized).](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=6350770)